lib/lorekeeper/json_logger.rb

```diff
@@ -95,6 +95,14 @@ def inspect
 
     private
 
+    # Some instrumentation libraries pollute the stacktrace and create a large output which may
+    # cause problems with certain logging backends.
+    # Hardcoring newrelic now here. In the future if this list grows, we may make it configurable.
+    def clean_backtrace!(backtrace)
+      backtrace.reject!{ |line| line.include?("newrelic_rpm") }
```

Comment:
is it safe to assume all the new_relic traces always stem from the gem directly? if so I think making this `"/newrelic_rpm-"` would be safer because who knows, someone might have some newrelic_rpm_helper module or something and they'd get filtered out.

Reply:
Always good to be safer. Updated the code.